bc_obps/service/operation_service_v2.py

```diff
@@ -98,3 +108,25 @@ def update_status(cls, user_guid: UUID, operation_id: UUID, status: Operation.St
         operation.save(update_fields=['status'])
         operation.set_create_or_update(user_guid)
         return operation
+
+    @classmethod
+    def create_or_replace_statutory_declaration(
+        cls, user_guid: UUID, payload: OperationStatutoryDeclarationIn
```

Comment:
Hello @Sepehr-Sobhani, any tips for optimizing this abomination? 😅 

<img width="331" alt="Screenshot 2024-08-15 at 3 42 58 PM" src="https://github.com/user-attachments/assets/f356a830-65e8-4f7a-9dd7-0a61bcdaaab9">

Reply:
As we are sending a large data-url to the backend, optimizing this endpoint requires us to adjust our approach to file uploads. Currently, we are sending the file using a base64-encoded value. To optimize this, we need to change it to binary format, and in this way, we can also use the built-in file upload functionality of Django-Ninja. https://django-ninja.dev/guides/input/file-params/
This could be a future issue :)